manifests/install/gem.pp

```diff
@@ -4,23 +4,19 @@
   $version,
 ) {
   require git
-  anchor{'r10k::ruby_done':}
+
   case $manage_ruby_dependency {
     'include': {
       include ruby
-      include ruby::dev
-      Class['::ruby']
-      -> Class['ruby::dev']
```

Comment:
Previously, this class needed to become the anchor resource.  By `require`ing it instead, we get the same effect.